### PR TITLE
no need to explicitly manage opened property of accordion on state ch…

### PR DESCRIPTION
**Pull request by mdulat**
…ange

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-accordion-collapse.js [hunk 1]
```diff
@@ -82,10 +80,9 @@ class ActivityAccordionCollapse extends SkeletonMixin(LitElement) {
 			<d2l-labs-accordion-collapse
 				flex
 				header-border
-				?opened=${this._isOpened()}
+				?opened=${this.hasErrors}
```
Comment (kieranderson) [inline components/d2l-activity-editor/d2l-activity-accordion-collapse.js:83]: This is just letting the accordion itself handle it's opening and closing and we only specifically opening it when he have an error, do I have that right? I'm not sure if we specifically set the opening/closing of accordion in other spots.

**Review by kieranderson — DISMISSED** [review]

File: components/d2l-activity-editor/d2l-activity-accordion-collapse.js (hunk 1 above)
Comment (mdulat) [reply components/d2l-activity-editor/d2l-activity-accordion-collapse.js:79]: That's correct. The only time we explicitly set it is when it has errors. If that changes in the future we can revisit this, but for now it was causing issues when it was explicitly setting that attribute whenever the accordion itself was changing it.

**Review by kieranderson — APPROVED** [review]

File: test/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-evaluation-editor.test.js [hunk 2]
```diff
@@ -176,11 +176,6 @@ describe('d2l-activity-assignment-editor-evaluation-editor', function() {
 			expect(summary.length).to.equal(4);
 		});
 
-		it('initializes as closed', async() => {
```
Comment (pureooze) [inline test/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-evaluation-editor.test.js:179]: Was this test not actually verifying that the accordion was closed on initial load? 😅

**Review by pureooze — APPROVED** [review]

File: test/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-evaluation-editor.test.js (hunk 2 above)
Comment (mdulat) [reply test/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-evaluation-editor.test.js:179]: It no longer has the `_opened` property to check, but the accordion wrapper checks whether it initializes as opened or closed properly https://github.com/BrightspaceHypermediaComponents/activities/blob/master/test/d2l-activity-editor/d2l-activity-accordion-collapse.js#L17

**Review by pureooze — APPROVED** [review]

**Pull request merged**